runtime/oti/j9nonbuilder.h

```diff
@@ -5766,6 +5766,7 @@ typedef struct J9JavaVM {
 	UDATA virtualThreadLinkNextOffset;
 	UDATA virtualThreadLinkPreviousOffset;
 	UDATA virtualThreadInspectorCountOffset;
+	J9VMEntryLocalStorage entryLocalStorage;
```

Comment:
What does this have to do with adding a DDR command?

Reply:
It doesn't seem reasonable at the VM level, and with nobody providing a value for this field it can't be useful.

Reply:
I think we should work towards a solution that doesn't depend on `J9JavaVM.entryLocalStorage`.